Src/IronPython/Compiler/Ast/TryStatement.cs

```diff
@@ -164,20 +170,23 @@ public class TryStatement : Statement {
             }
             
             return Ast.Block(
-                GetVariables(lineUpdated, runElse), 
+                GetVariables(lineUpdated, runElse, previousExceptionContext),
                 AddFinally(result),
                 AstUtils.Default(typeof(void))
             );
         }
 
-        private static ReadOnlyCollectionBuilder<MSAst.ParameterExpression> GetVariables(MSAst.ParameterExpression lineUpdated, MSAst.ParameterExpression runElse) {
+        private static ReadOnlyCollectionBuilder<MSAst.ParameterExpression> GetVariables(MSAst.ParameterExpression lineUpdated, MSAst.ParameterExpression runElse, MSAst.ParameterExpression previousExceptionContext) {
```

Comment:
There's gotta be utility for this...

Reply:
No idea, could be something like:
```C#
private static ReadOnlyCollectionBuilder<MSAst.ParameterExpression> GetVariables(params MSAst.ParameterExpression[] expr)
    => new ReadOnlyCollectionBuilder<MSAst.ParameterExpression>(expr.Where(e => e != null));

```

Reply:
Would this belong in `AstUtils`?

Reply:
I'm not really seeing a use case outside of `TryStatement` so probably not worth it.